src/scripts/browserdeviceprofile.js

```diff
@@ -214,6 +214,14 @@ define(['browser'], function (browser) {
                 break;
             case 'avi':
                 supported = browser.tizen || browser.orsay || browser.web0s || browser.edgeUwp;
+                // New Samsung TV don't support XviD/DivX
+                if (browser.tizen) {
+                    videoCodecs.push('h264');
+                    if (canPlayH265(videoTestElement, options)) {
+                        videoCodecs.push('h265');
+                        videoCodecs.push('hevc');
+                    }
+                }
```

Comment:
Why not just remove Tizen from the `supported` variable?

Reply:
Because AVI can contain other things than XVID, like H.264 and H.265.

No need to transcode an entire container because one codec it can use isn't supported.

Reply:
Wouldn't we still need to report false or something so the regular behavior isn't enabled for this case at the end of the function?

Reply:
As-is, I believe it will report:

```
Container: "avi"
Type: 'Video',
VideoCodec: "h264,h265,hevc"
AudioCodec: ""
```

Which should have the desired effect. Hopefully it's tested, though.

Reply:
it supported H.264 in `avi`. H.265 and HEVC possibly but not tested

Reply:
I would add a comment at least to explain the solution.